packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeEthForm/UnstakeEthForm.tsx

```diff
@@ -91,14 +91,6 @@ export const UnstakeEthForm = () => {
 
             <Options symbol={symbol} />
 
-            <WarningsWrapper>
```

Comment:
By this change, this banner will be gone for all options :( 
![Screenshot 2024-08-19 at 17 42 57](https://github.com/user-attachments/assets/eb56f53b-5266-413e-992a-9c53de2b1f3f)

Reply:
this should work https://github.com/trezor/trezor-suite/pull/13873/commits/7b88b68de0fe801bb4535e9a8a29de66075619be

Reply:
looks good